lib/plugins/aws/package/lib/mergeIamTemplates.js

```diff
@@ -72,6 +72,14 @@ module.exports = {
     iamRoleLambdaExecutionTemplate.Properties.Path = this.provider.naming.getRolePath();
     iamRoleLambdaExecutionTemplate.Properties.RoleName = this.provider.naming.getRoleName();
 
+    // set role tags
+    if (iamRole.tags) {
+      iamRoleLambdaExecutionTemplate.Properties.Tags = Object.keys(iamRole.tags).map((key) => ({
```

Comment:
I noticed at least 5 instances of people doing this kind of stuff across codebase - what's your view on making a small lib for this? Pretty basic code, unsure if it's a worthwhile.

Reply:
I think that while it might make sense, I think it could be submitted as a separate refactoring PR/commit instead of adding it to this one